### DISCUSSION TITLE
Hydration on .60 ui?

### DISCUSSION THREAD

**Discussion by saona-raimundo** [slint-ui/slint#821] [Q&A] [answered]
We can build `.60` ui at run time using [`sixtyfps_interpreter`](https://docs.rs/sixtyfps-interpreter/0.1.5/sixtyfps_interpreter/). Also, @tronical mentioned in #171 that
> SixtyFPS creates an item tree at compile time that’s equivalent to the scene graph node tree. It’s traversed and femtovg generates list of GL commands (draw, geometry buffers and minimal texture).

So my question is:
Is hydration possible? or is it planned?

By hydration I mean the possibility to update a component of the ui with out rendering the whole ui again.

**Comment by tronical** [comment, marked as answer]
We plan on supporting this in the future. We think they we will need it for the software renderer to be efficient.
